File: data-service/spring-data-provider/src/main/java/org/aykhan/dataprovider/dto/comments/CommentResponse.java
package org.aykhan.dataprovider.dto.comments;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.aykhan.dataprovider.dto.user.UserResponse;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class CommentResponse {
  private Long postId;
  private Long commentId;
  private String body;
  private UserResponse author;
}

File: data-service/scala-sql-dataprovider/app/dtos/comment/CommentResponse.scala
package dtos.comment

import dtos.user.UserResponseDTO
import play.api.libs.json.Json

case class CommentResponse(postId: Long,
                           commentId: Long,
                           body: String,
                           author: UserResponseDTO)

object CommentResponse {
  implicit val writes = Json.writes[CommentResponse]
}
